src/classes/PMT_Payment_TDTM.cls

```diff
@@ -201,4 +162,132 @@ public class PMT_Payment_TDTM extends TDTM_Runnable {
         }
     }
 
+    /*******************************************************************************************************
+    * @description          Retrieve a Id Map of Opportunties related to a Set of Ids
+    * @param oppIds         Set of Opportunity Ids to retrieve
+    * @return Map<Id, Opp>  This method returns a map of Id to opportunity
+    ********************************************************************************************************/
+    private Map<Id, Opportunity> getPaymentOpportunities(Set<Id> oppIds) {
+        String soql = 'SELECT Id, Amount, IsClosed, IsWon, StageName, ';
+        if (isCurrencyFieldAvailable) {
```

Comment:
I don't think this check is necessary since getPaymentOpportunities will only get called method if `isCurrencyFieldAvailable == true`. Unless there's issues with testing?

Reply:
@seanpat09, Yeah it's an issue with the testing.  originally I got rid of it then I noticed that we have a mock util currency that sets it to true and so on non-multicurrency orgs the test would fail - Not sure if there's a better way to handle it.